src/foam/dao/ManyToManyRelationshipDAO.js

```diff
@@ -28,13 +28,14 @@ foam.CLASS({
     'junctionProperty',
     'joinDAOKey',
     'targetProperty',
+    'dest',
```

Comment:
I know I recommend "dest" first, but now that I think about it, "junctionCls" would be a more appropriate name.